### PR TITLE
added optional message param to the Title Should Be function

### PR DESCRIPTION
**Pull request by rubygeek**
Added code and tests to complete #1042

### CODE REVIEW

**Comment by aaltat** [comment]
We lately have had problems with Travis and it looks like you suffered from the same problem. I did restart the Travis jobs and let's see how it goes.

File: src/SeleniumLibrary/keywords/browsermanagement.py [hunk 1]
```diff
@@ -303,12 +303,17 @@ def log_title(self):
         return title
 
     @keyword
-    def title_should_be(self, title):
-        """Verifies that current page title equals ``title``."""
+    def title_should_be(self, title, message=None):
+        """Verifies that current page title equals ``title``.
+
+        The ``message`` argument can be used to override the default error
```
Comment (aaltat) [inline src/SeleniumLibrary/keywords/browsermanagement.py:309]: Please add at the last line that `message` argument is new in SeleniumLibrary 3.1.

**Comment by aaltat** [comment]
There was one tiny thing to add. Also if you want to have credit from your work, please add your name and what did you do in the https://github.com/robotframework/SeleniumLibrary/blob/master/CHANGES.rst

**Comment by pekkaklarck** [comment]
Thanks for PR!

Docs should have a note that this is new in SL 3.1 (or whatever the version is).

**Review by aaltat — APPROVED** [review]

**Comment by aaltat** [comment]
Looking good, merged to master. Thanks for the contribution.

**Pull request merged**